cmd/self-signer/rotate.go

```diff
@@ -30,10 +31,62 @@ var rotateCmd = &cobra.Command{
 	Run:   rotate,
 }
 
+var (
+	clientFlag, caFlag, nodeFlag bool
+	caCron, nodeAndClientCron    string
+	readinessWait                string
+)
+
 func init() {
 	rootCmd.AddCommand(rotateCmd)
+
+	rotateCmd.Flags().BoolVar(&clientFlag, "client", false, "if set rotates client certificate")
+	rotateCmd.Flags().BoolVar(&nodeFlag, "node", false, "if set rotates node certificate")
+	rotateCmd.Flags().BoolVar(&caFlag, "ca", false, "if set rotates ca certificate")
+
+	rotateCmd.Flags().StringVar(&caCron, "ca-cron", "", "cron of the CA certificate rotation cron")
+	rotateCmd.Flags().StringVar(&nodeAndClientCron, "node-client-cron", "", "cron of the node and client certificate rotation cron")
+
+	rotateCmd.Flags().StringVar(&readinessWait, "readiness-wait", "30s", "readiness wait for each replica of crdb cluster")
+
 }
 
 func rotate(cmd *cobra.Command, args []string) {
-	fmt.Println("rotate called")
+	if clientFlag && nodeFlag && caFlag {
```

Comment:
Doesn't this need to be 
`(clientFlag || nodeFlag) && caFlag` otherwise client and CA could be rotated together.

Reply:
Other cert rotation will be skipped if `caFlag` is set, so client and CA won't be rotated together. But you are right we should handle above case and error out instead.
Fixed